application/src/main/java/org/togetherjava/tjbot/commands/filesharing/DeleteGistCommand.java

```diff
@@ -0,0 +1,57 @@
+package org.togetherjava.tjbot.commands.filesharing;
+
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+import org.togetherjava.tjbot.config.Config;
+
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+public final class DeleteGistCommand extends SlashCommandAdapter {
+
+    private static final Logger logger = LoggerFactory.getLogger(DeleteGistCommand.class);
+
+    private static final String MESSAGE_ID_OPTION = "message-id";
+
+    public DeleteGistCommand(@NotNull Config config) {
+        super("remove-gist", "Deletes gist's from auto file-sharing feature!",
```

Comment:
the description sounds more like a description for a programmer and less for an actual user. `"Delete a previously uploaded gist"`

Reply:
isn't this file is deleted now?

Reply:
yea should be
it was WIP